BookStore.iml

```diff
@@ -51,6 +51,15 @@
     <orderEntry type="library" name="Maven: org.unbescape:unbescape:1.1.6.RELEASE" level="project" />
     <orderEntry type="library" name="Maven: org.slf4j:slf4j-api:1.7.29" level="project" />
     <orderEntry type="library" name="Maven: org.thymeleaf.extras:thymeleaf-extras-java8time:3.0.4.RELEASE" level="project" />
+    <orderEntry type="library" scope="TEST" name="Maven: org.springframework.security:spring-security-test:5.2.1.RELEASE" level="project" />
```

Comment:
also - should this be committed?

Reply:
never mind, this is because these files are out of date - I realized this because every time I locally pull a new branch I have to reimport the maven project, and it causes theses changes

Reply:
maven updated it automatically, so it should be there as far as I know